backend/logic/printful/sync/stockUpdate.js

```diff
@@ -0,0 +1,61 @@
+const get = require('lodash/get')
+const { decryptConfig } = require('../../../utils/encryptedConfig')
+
+const { Product } = require('../../../models')
+const { getLogger } = require('../../../utils/logger')
+
+const log = getLogger('utils.printful.writeProductData')
+
+/**
+ * Appends a productId to a collection.
```

Comment:
This description seems incorrect, right? This method should only update stockLeft in the Product table?

Reply:
My bad, I will update

Reply:
Updated